src/app_registry/web.py

```diff
@@ -105,23 +119,31 @@ def build_from_config(config: Config, validate: bool = True):
     )
 
     # Validate the app registry data against the provided schemas.
-    if validate:
+    if validate_input:
         data.validate(schemas)
 
     # Generate the aggregated apps metadata registry.
     apps_meta = generate_apps_meta(
-        data=data, schema=schemas.apps_meta if validate else None
+        data=data, schema=schemas.apps_meta if validate_output else None
     )
 
+    root = Path(config.build.html)
+
     # Remove previous build (if present).
-    shutil.rmtree(Path(config.build.html), ignore_errors=True)
+    shutil.rmtree(root, ignore_errors=True)
 
     # Copy static data (if configured).
     if config.build.static_src:
-        shutil.copytree(config.build.static_src, Path(config.build.html))
-
-    # Build the html pages.
-    build_html(apps_meta, dest=Path(config.build.html))
+        shutil.copytree(config.build.static_src, root)
+
+    for outfile in chain(
+        # Build the html pages.
+        build_html(apps_meta, root=root),
+        # Build the API endpoints.
+        build_api_v1(apps_meta, base_path=root),
+        build_api_v2(apps_meta, base_path=root / "api" / "v2"),
```

Comment:
```suggestion
        build_api_v0(apps_meta, base_path=root),
        build_api_v1(apps_meta, base_path=root / "api" / "v1"),
```

Reply:
I think we should consider to label the "first" API with version number 0 and the new one with version number 1.